### ISSUE TITLE
Bond Orders

### ISSUE THREAD

**Issue by dgasmith** [openforcefield/qca-dataset-submission#75]
At the moment we compute and store both Wiberg and Mayer bond orders. It would be best to pick a single index to store to help limit total stored data and data pulled on each query. I believe @ChayaSt will be analyzing the latest geometry optimization dataset to determine which is the best to store.

**Comment by ChayaSt** [comment]
@dgasmith, After discussing this with other OFF members, we decided to see if we can keep both indices, but only for the optimized geometries. 

Another option might be to only keep indices that are above 0.001 to limit the amount of data stored. 

The question is, if the WBOs change that the connectivity changed in a torsion scan, would it be possible to somehow invalidate the rest of the molecules in the minimization trajectories? 

pinging @jchodera and @j-wags on this.

**Comment by dgasmith** [comment]
Sure, we can continue to store both.

What do you mean by invalidate?

**Comment by j-wags** [comment]
We have the idea to "invalidate" final molecules by setting a bool (maybe `connectivity_changed`) in their metadata. In the short term, we could simply handle this on the client-side, by comparing their final connectivity to the stored CMILES. In the long term, our aspirational "crawler" could automatically iterate over the final molecules, and set a `connectivity_changed` bool as metadata on the final molecule.

TL;DR -- Just storing the bond orders is fine now, we don't need to worry about "invalidating" in the short term. Storing bond orders sparsely with a cutoff of 0.001 is perfect for us.

**Comment by dgasmith** [comment]
Yup, crawler is the way to go. Setting up automatic triggers with QCFractal based on results is do-able as well, but would represent a long-term infrastructure investment. Good to validate sparsity we will incorporate this into the schema when we can.

**Comment by j-wags** [comment]
Superseded by the existence of the QCSubmit connectivity change filter.
